AssistantV1.playground/Pages/Mentions.xcplaygroundpage/Contents.swift

```diff
@@ -1,42 +1,10 @@
 //:## Mentions
 
-import PlaygroundSupport
-
-// Enable support for asynchronous completion handlers
-PlaygroundPage.current.needsIndefiniteExecution = true
-
 import AssistantV1
 
 let assistant = setupAssistantV1()
-let workspaceID = WatsonCredentials.AssistantV1Workspace
-
-// Setup
-
-assistant.createEntity(
-    workspaceID: workspaceID,
-    entity: "beverage",
-    values: [
-        CreateValue(value: "water"),
-        CreateValue(value: "orange juice"),
-        CreateValue(value: "soda")
-    ])
-{
-    response, error in
-
-    guard let entity = response?.result else {
-        print(error?.localizedDescription ?? "unknown error")
-        return
-    }
-}
-
-assistant.createIntent(workspaceID: workspaceID, intent: "place_order", examples: [CreateExample(text: "Can I get a soda?")]){
-    response, error in
-
-    guard let intent = response?.result else {
-        print(error?.localizedDescription ?? "unknown error")
-        return
-    }
-}
+let workspaceID = getWorkspaceID()
+createSampleEntity(workspaceID: workspaceID)
```

Comment:
Don't you also need to call `createSampleIntent()` and `deleteSampleIntent()`?

Reply:
I don't think these are needed.  I originally added these because I was trying to figure out how to set up the entity to have "mentions" -- so that the return value would look like the example response in the API ref.  But it turns out that you can't create "mentions" via the API -- only with the tooling ☹️.  So there is no need for the sample intent on this page.

Reply:
Well that's lame. So the `createIntent` step that used to be there wasn't actually doing anything?

Reply:
My memory is fuzzy on this, but I can't see any need for the sample intent in this flow.